CHANGELOG.md

```diff
@@ -8,6 +8,8 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 ## [Unreleased]
 
 - Improves caching strategy and use new scope convention to declare this a core Heroku CLI plugin. 
+- Added command `pipelines:connect` to connect an existing pipeline to a Github repo.
+- Added command `pipelines:reviewapps` to enable|disable review apps with an existing pipeline.
```

Comment:
This was modified in code, but not here. Updated with this [commit](https://github.com/heroku/heroku-pipelines/commit/6b9a04c157ccaedfc3eec94722deb1ae8f40474a).

Reply:
Also, +1 on what @jdxcode said https://github.com/heroku/cli/pull/767#issuecomment-376566691.